messytables/commas.py

```diff
@@ -46,18 +46,19 @@ def __init__(self, f, encoding):
     def __iter__(self):
         return self
 
-    def next(self):
+    def __next__(self):
         line = self.reader.readline()
         if not line or line == '\0':
             raise StopIteration
         result = line.encode("utf-8")
         return result
 
+    next = __next__
 
 def to_unicode_or_bust(obj, encoding='utf-8'):
-    if isinstance(obj, basestring):
-        if not isinstance(obj, unicode):
-            obj = unicode(obj, encoding)
+    if PY2:
```

Comment:
This function doesn't seem to do anything at all under Python3. Is that intended?